elasticdl/python/tests/servicer_test.py

```diff
@@ -215,7 +217,7 @@ def makeGrad():
 
         # Report a current version with part of gradients, should be accepted.
         req = makeGrad()
-        del req.gradient["y"]
+        req.gradient.pop(-1)
```

Comment:
If the index of pop is not given, the last element will be removed. So just `pop()` will be fine.

Reply:
Done